custom/conf/app.ini.sample

```diff
@@ -116,6 +116,8 @@ DEFAULT_THEME = gitea
 THEMES = gitea,arc-green
 ; Whether the full name of the users should be shown where possible. If the full name isn't set, the username will be used.
 DEFAULT_SHOW_FULL_NAME = false
+; Whether to search within description at repository search on explore page.
+SEARCH_REPO_DESCRIPTION = false
```

Comment:
I know that I was not the one worried about performance in this case, but I think `SEARCH_REPO_DESCRIPTION` should default to true.  @sapk, @lunny

Reply:
I also agree that this should default to true

Reply:
ok for me

Reply:
I set it to false to not change behavior for existing installations, but I think most will want to have it on. @lunny Your opinion on this?

Reply:
It does change behavior but we tend to enable new features by default if only they have dramatic change in performance/disk usage. I don't think this have such big impact on performance

Reply:
Changed to default true!